base/strings/search.jl

```diff
@@ -259,7 +259,7 @@ findnext(t::AbstractString, s::AbstractString, i::Integer) = _search(s, t, i)
     findlast(pattern::AbstractString, string::AbstractString)
 
 Find the last occurrence of `pattern` in `string`. Equivalent to
-[`findlast(pattern, string, lastindex(s))`](@ref).
+[`findprev(pattern, string, lastindex(s))`](@ref).
```

Comment:
Would also be a good idea to update to `lastindex(string)` instead, to match the signature on the lines above:

```suggestion
[`findprev(pattern, string, lastindex(string))`](@ref).
```
alternatively change `string` to `s` on L259.